runetree.go

```diff
@@ -152,6 +171,20 @@ func NewRuneTree() *RuneNode {
 	}
 }
 
+// ContractionsTree creates a specialized RuneTree for handling contractions
+func ContractionsTree() *RuneNode {
```

Comment:
Slight nitpick, but the wording of this function being `ContractionsTree` sounds like it ought to be a struct, and it's a bit awkward that it takes no arguments. It maybe would warrant its own struct with a pointer to a `RuneNode` struct, and this function could be called `NewContractionsTreeFromDefaults` or something. I don't understand the underlying implementation here though so take this with a grain of salt. Just purely from an idioms perspective. Go's function naming idioms are fairly specific, which is why this pops out.